### DISCUSSION TITLE
Multiple ID/Relay servers

### DISCUSSION THREAD

**Discussion by mateiandrei94** [rustdesk/rustdesk#7236] [Q&A] [answered]
Given recent outages on public servers, I have decided to host my own for work.
Now I find myself in a situation where I'm unable to connect to anyone that uses the public servers and vice versa.
I have to either configure my own client to use the public servers or instruct someone non technical to configure their rust desk client to use my servers.
My feature request involves having support in client for multiple servers.
It would be nice to have, and be able to configure, the official client for use in scenarios where you may want people on different [ID/Relay server] including the public ones be able to connect to your machine, as well as being able to connect to different machines that are only configured to a self hosted [ID/Relay server]

Visually this is what i mean:
- I have my rust desk installation and it's connected to both the public servers and a self hosted one.
- for each server i have a different id and password pair
- given that i have multiple servers configured, i now have to specify the server when i want to connect to someone
- my recent sessions also display in square brackets the server on which a host can be found.

![image](https://github.com/rustdesk/rustdesk/assets/31540942/e86cc2d5-ea8c-4204-be7b-427bf0d8111f)

**Comment by RicRey1988** [comment]
It seems to be an excellent idea, I hope it is received and can be applied.

**Comment by c-michail** [comment]
Please implement this!

**Comment by Joaquim3** [comment]
I did this SH workaround. It's a bit dirty. But it's working fine. Hope this can help.
Check the attached TXT file.
[rustdesk.txt](https://github.com/user-attachments/files/15935726/rustdesk.txt)

**Comment by rustdesk** [comment, marked as answer]
close to favor https://github.com/rustdesk/rustdesk/discussions/1258
